engine/orchestration/src/main/java/com/cloud/vm/VirtualMachineManagerImpl.java

```diff
@@ -1112,6 +1110,13 @@ public void orchestrateStart(final String vmUuid, final Map<VirtualMachineProfil
 
                     vmGuru.finalizeDeployment(cmds, vmProfile, dest, ctx);
 
+                    // Get VM extraConfig from DB and set to VM TO
+                    UserVmDetailVO detailVO = userVmDetailsDao.findDetail(vm.getId(), "extraConfig");
```

Comment:
How about rename the variable to `extraConfigVO`? Also, I think we can use an static variable the detail name instead of using the string "extraConfig" each time we want to access it

Reply:
Removed all instances of "extraConfig" and replaces with ApiConstants.EXTRA_CONFIG.
Also renamed detailVO to extraConfigDetail